packages/client/jest.config.js

```diff
@@ -3,14 +3,14 @@ const tsJestPresets = require('ts-jest/presets')
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  globals: {
-    'ts-jest': {
-      diagnostics: false
-    }
-  },
   transform: {
-    '\\.(gql|graphql)$': 'jest-transform-graphql',
-    ...tsJestPresets.jsWithBabel.transform
+    '\\.(gql|graphql)$': '../server/__tests__/jest-transform-graphql-shim.js',
+    '^.+\\.tsx?$': [
```

Comment:
copied from https://kulshekhar.github.io/ts-jest/docs/getting-started/options/

Reply:
+1 Should we remove the `preset` as the warning box on the page linked suggests?

Reply:
good catch! fixed